src/olympia/reviewers/templates/reviewers/emails/held_for_review.ltxt

```diff
@@ -1,7 +1,16 @@
 {% extends "reviewers/emails/base.ltxt" %}{% block content %}
-Thank you for submitting your add-on {{ name }} {{ version }} to addons.mozilla.org. Your submission is taking longer than usual to be reviewed and signed. This process could take a few extra hours and up to a couple of days. Due to disruptions caused by the COVID-19 pandemic, it may take up to a week.
+Hello,
 
```

Comment:
The greeting is already in the base template. Please remove it (and the blank lines) to resolve the duplication.